### DISCUSSION TITLE
Ability to hide/disable "Documents" and "Backlinks" section?

### DISCUSSION THREAD

**Discussion by keithfawcett** [outline/outline#6890] [General]
Is there a way to disable this section?

![Screenshot 2024-05-10 081641](https://github.com/outline/outline/assets/592055/2b493161-783c-404b-b41c-26ab82f01329)

**Comment by tommoor** [comment]
There isn't currently, it hasn't been asked before – any particular reason they are not wanted? It's generally useful info.

**Comment by keithfawcett** [comment]
In this instance I already created a numbered list with links to the same
nested docs, so having the items at the bottom are redundant.
